### DISCUSSION TITLE
Xrspatial download issue

### DISCUSSION THREAD

**Discussion by Adriannaluger** [earthlab-education/Earth-Analytics-2023-01-Intro#673] [! GitHub help]
Hi! For some reason when attempting to get xrspatial into my codespace it was unsuccessful. In class we tried changing directories with no success. Potentially because my devcontainer is not in my notebooks folder? What do you recommend? Someone mentioned that deleting the entire codespace and creating a new one may be best, however, since I am having commit/sync issues I don't exactly know how to navigate this... let me know if you think this is even worth troubleshooting or if pseudo code is best here. Sorry for a million questions haha... thanks!! @eculler

**Comment by NobodyLivesHere** [comment]
If the codespace was built before xspatial was put into the docker file (devcontainer) then it's possible that it's just trying to reference an outdated image. You should not need to delete the whole codespace, you can just rebuild the container. 

https://docs.github.com/en/codespaces/developing-in-a-codespace/rebuilding-the-container-in-a-codespace

That will show you how to do it.  Deleting and  remaking the codespace will achieve the same end
